distributed/utils.py

```diff
@@ -1384,7 +1384,7 @@ def reset_logger_locks():
             isinstance(traitlets.config.Application.instance(), NotebookApp)
         )
 
-    if not jupyter_event_loop_initialized:
+    if not jupyter_event_loop_initialized and tornado.version_info[0] >= 5:
```

Comment:
If the previous code block is only necessary for this if statement it might make more sense to the tornado version check on 1375 with the asyncio check .

Reply:
Yes, that's the case.  You're right that it's probably better to include this check above.  It's fine either way though.